ha/setup/ha_setup.py

```diff
@@ -858,6 +857,10 @@ def process(self):
                 self._confstore.delete(f"{const.PVTFQDN_TO_NODEID_KEY}/{node_name}")
             # Delete the config file
             self.remove_config_files()
+
+            Log.info("Post_install being called from cleanup command")
+            PostInstallCmd.process(self)
```

Comment:
Is this tested? How can a member function be called like this?

Reply:
yes, Tested. it works fine
2021-07-07 07:16:19 ha_setup INFO [process] Post_install being called from cleanup command
2021-07-07 07:16:19 ha_setup INFO [process] Processing post_install command
2021-07-07 07:16:19 ha_setup INFO [process] cleanup: Copied HA configs file.
2021-07-07 07:16:19 ha_setup INFO [process] Found required cluster packages installed.
2021-07-07 07:16:30 ha_setup INFO [process] hacluster is a part of the haclient group
2021-07-07 07:16:30 ha_setup INFO [process] post_install command is successful
2021-07-07 07:16:30 ha_setup INFO [process] cleanup command is successful

Reply:
It works but this is not the correct way instance method. process() method is not using any class attribute. This is the reason, there is no problem. If it start using those tomorrow, then the same code will break. Please make the changes.